packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -108,23 +96,7 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
       <div {...divProps} className={classNames.coin}>
```

Comment:
It looks like this component is still a few levels deep. For the scenario in which we need just the image or the initials, can we make it one div? Or create a separate component that's just one div?

Reply:
Yes, like https://codepen.io/dzearing/pen/BVjzRW

Reply:
Yep, i basically put the PR here to have screener run on my commits, so i can follow i'm not really breaking anything.

I'll pick this up today and see if i can reduce the DOM impact it further :)

Reply:
Okay please let me know how I can help!